### ISSUE TITLE
[1.10.2 - 0.10-58] Garden Cloche Redstone Interaction (+1 more)

### ISSUE THREAD

**Issue by VallenFrostweaver** [BluSunrize/ImmersiveEngineering#2122]
Currently using Redstone Wire Connectors (on any side of a Cloche) set to output connected to the another redstone wire connector that is set to input and a lever being used has no results. Setting redstone lamps near any of the connections shows a redstone signal is being sent but the Cloche still continues functioning. Setting a lever on/next to the Cloche and turning it off will stop the Cloche.  Trailing Redstone to a cloche will stop it if connected to a lever in the OFF position.

In conclusion, I am seeing the redstone output from redstone wire connectors being ignored by the cloche.

Second small issue - sugar cane doesn't grow on dirt or grass in the cloche but will on sand/red sand.

**Comment by Sabbatha** [comment]
I'm seeing the same thing. I am able to turn on a lamp with my redstone wire connectors and the cloche shuts off with a lever. However, the exact same setup with a restone wire connector does nothing to shut off the cloche.

**Comment by BluSunrize** [comment]
1.10 is out of support.

**Comment by Sabbatha** [comment]
I got it to work by placing a block next to the cloche and powering that with the redstone wire connector, for anyone else running into this on this version.
